<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: black;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>120</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#backButton{
	border-width: 2px;
	border-style: none;
	border-color: white;
	border-radius: 10px;
	background-color: rgba(110, 126, 141, 255);
	color: white;
}

QPushButton#backButton:pressed{
	padding-left: 2px;
	padding-top: 2px;
	background-color: rgba(110, 126, 141, 200);
}
</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startTimer">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>200</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgba(122, 163, 122, 255);
	color: white;
	border-style: solid;
	border-color: white;
	border-radius: 10px;
	border-width: 2px;
}

QPushButton:pressed{
	padding-left: 5px;
	padding-top: 5px;
	background-color:  rgba(122, 163, 122, 200);
}
</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>400</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	border: 2px solid white;
	background-color: transparent;
	border-radius:5px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>400</width>
       <height>120</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QLCDNumber{
	border: white;
	background-color: transparent;
	color: white;
	border-radius: 5px;
}</string>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>8</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>670</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	border: 2px solid white;
	background-color: transparent;
	border-radius: 10px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Leelawadee UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border: 0px black none;
	color: white;
	background-color: transparent;
}</string>
     </property>
     <property name="text">
      <string>Hours</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QDial" name="timerDialSecs">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>40</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QDial{
	background: transparent;
}</string>
     </property>
     <property name="maximum">
      <number>59</number>
     </property>
    </widget>
    <widget class="QDial" name="timerDialMins">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QDial{
	background: transparent;
}</string>
     </property>
     <property name="maximum">
      <number>59</number>
     </property>
    </widget>
    <widget class="QDial" name="timerDialHours">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QDial{
	background: transparent;
}

QDial::handle{
	color: red;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Leelawadee UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border: 0px black none;
	color: white;
	background-color: transparent;
}</string>
     </property>
     <property name="text">
      <string>Minutes</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>10</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Leelawadee UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border: 0px black none;
	color: white;
	background-color: transparent;
}</string>
     </property>
     <property name="text">
      <string>Seconds</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="resetTimer">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>490</y>
      <width>200</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-width: 2px;
	border-style: solid;
	border-color: white;
	border-radius: 10px;
	background-color: rgba(110, 126, 141, 255);
	color: white;
}

QPushButton:pressed{
	padding-left: 2px;
	padding-top: 2px;
	background-color: rgba(110, 126, 141, 200);
}
</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pomoTimer">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>490</y>
      <width>200</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: white;
	color: black;
	border-style: none;
	border-color: black;
	border-radius: 10px;
	border-width: 5px;
}

QPushButton:pressed{
	padding-left: 5px;
	padding-top: 5px;
	background-color: white;
}
</string>
    </property>
    <property name="text">
     <string>Pomodoro</string>
    </property>
   </widget>
   <zorder>frame_2</zorder>
   <zorder>backButton</zorder>
   <zorder>startTimer</zorder>
   <zorder>frame</zorder>
   <zorder>resetTimer</zorder>
   <zorder>pomoTimer</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
